modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -12,4 +12,10 @@ Only use this procedure for {ibm-z-name} clusters.
 Depending on your {ibm-z-name} environment, you can choose from the following options:
 
 * Adding {ibm-z-name} agents with z/VM
-* Adding {ibm-z-name} agents with {op-system-base} KVM
\ No newline at end of file
+* Adding {ibm-z-name} agents with {op-system-base} KVM
+* Adding {ibm-z-name} agents with Logical Partition (LPAR)
+
+[NOTE]
+====
+Currently, ISO boot support on {ibm-z-name} (s390x) is available only for RHEL KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and LPAR, only PXE boot is supported.
```

Comment:
```suggestion
Currently, ISO boot support on {ibm-z-name} (`s390x`) is available only for RHEL KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and LPAR, only PXE boot is supported.
```

An attribute exists for RHEL:

![Screenshot from 2024-08-14 10-37-24](https://github.com/user-attachments/assets/c55420b8-6e90-4939-af73-ea5a6734ae04)

Reply:
As LPAR is mentioned in a new document, please state " Logical Partition (LPAR).